approval_polls/models.py

```diff
@@ -1,52 +1,103 @@
+from django.core.urlresolvers import reverse
 from django.db import models
 from django.contrib.auth.models import User
 
+from django.utils import timezone
+
+
 class Poll(models.Model):
-  question = models.CharField(max_length=200)
-  pub_date = models.DateTimeField('date published')
-  user = models.ForeignKey(User)
+    '''A Poll class. This is the top level container for a Poll.
+
+    referenced by Choices model and the Ballot model
+    '''
+
+    question = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
+    user = models.ForeignKey(User)
+    open_date = models.DateField()
+    close_date = models.DateField()
+    suspended = models.BooleanField(default=False)
+
+    def is_closed(self):
+        '''Determine if the poll is open or closed
+
+        Arguments:
+            None
+
+        Returns:
+            boolean: True if the close_date < datetime.datetime.now()
```

Comment:
Yes, that makes sense. It would be useful to have auto generated documentation at some point.